modulegen/cmd/new/example.go

```diff
@@ -0,0 +1,26 @@
+package new
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/testcontainers/testcontainers-go/modulegen/internal"
+	"github.com/testcontainers/testcontainers-go/modulegen/internal/model"
+)
+
+func NewExampleCmd() *cobra.Command {
+	exampleVar := &model.ExampleVar{}
+	cmd := &cobra.Command{
+		Use:   "example",
+		Short: "Create a new Example",
+		Long:  "Create a new Example",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return internal.Generate(*exampleVar, false)
+		},
+	}
+	cmd.Flags().StringVarP(&exampleVar.Name, "name", "n", "", "Name of the example. Only alphabetical characters are allowed.")
+	cmd.Flags().StringVarP(&exampleVar.NameTitle, "title", "t", "", "(Optional) Title of the example name, used to override the name in the case of mixed casing (Mongodb -> MongoDB). Use camel-case when needed. Only alphabetical characters are allowed.")
+	cmd.Flags().StringVarP(&exampleVar.Image, "image", "i", "", "Fully-qualified name of the Docker image to be used by the example")
+	_ = cmd.MarkFlagRequired("image")
+	_ = cmd.MarkFlagRequired("name")
```

Comment:
Have you considered defining those flags, that are reused in the sibling `module` subcommand, into the `new` subcommand as PersistentFlags?

They will be shared downstream to all the subcommands of the command that defined them.

https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#persistent-flags

Reply:
I did but then I wouldn't be able to use a different description . I mean example or module depending on the subcommand